runtime/convert.go

```diff
@@ -16,16 +17,57 @@ func String(val string) (string, error) {
 	return val, nil
 }
 
+// StringSlice converts 'val' where individual strings are separated by
+// 'sep' into a string slice.
+func StringSlice(val, sep string) ([]string, error) {
+	s := strings.Split(val, sep)
+	values := make([]string, len(s))
+	for i, v := range s {
+		values[i] = v
+	}
+	return values, nil
```

Comment:
Why are you creating a new slice? Just `return strings.Split(val, sep), nil`

Reply:
Good point.. copy pasta of the other slice conversion functions. Will change.